code/index.html

```diff
@@ -4,27 +4,127 @@
+        <input type="email" id="email" name="email" style="width: 150px;" placeholder="Your email" required>
+        <br><br>
+       <!--RADIO BUTTONS-->
+    <div class="boxes">   
+      <div class="radio-buttons">
+        <input type="radio" name="gender" value="male" required>Male <br>
+        <input type="radio" name="gender" value="female" required>Female <br>
+        <input type="radio" name="gender" value="other" required>Other <br><br>
       </div>
+      <!--CHECKBOXES-->
+      <div class="checkboxes">
+        <p class="checkbox-text">I'm interested in...</p>
+        <input type="checkbox" name="group-training" id="group-training" value="group-traning">
+        <label for="group-traning">Group training</label>
+        <br>
+        <input type="checkbox" name="PT" id="PT" value="PT">
+        <label for="PT">PT</label>
+        <br>
+        <input type="checkbox" name="diet" id="diet" value="diet">
+        <label for="diet">Diet</label>
+        <br><br>
+      </div>
+    </div> 
+
+        <input type="submit" value="Try it out!">
+    </form>
 
-      <div class="question" id="section2">Section 2</div>
+      <!-- 5. Accordion -->
+    <section class="accordion">
+      <h1 class="FAQ">FAQ</h1>
+      <div class="question" id="section1">Sounds like fun! Can I try?</div>
+      <div class="answer">
+          <p>Of course! You are more than welcome to contact us and sign up for a free, but sweaty workout!</p>
+      </div>
+          <br> 
+      <div class="question" id="section2">Do I have to be really fit to keep up?</div>
       <div class="answer">
-        <p>Lorem ipsum...</p>
+        <p>No, not allt all! All of our WOD's (Workout Of the Day) have different levels, so everyone can participate to the best of their ability. </p>
       </div>
+      <br>
+      <div class="question" id="section3">Do I need some special eqipment??</div>
+      <div class="answer">
+        <p>No, your usual workout clothes and shoes are fine!</p>
+      </div>
+      <br>
+      <div class="question" id="section4">What is a"box"?</div>
+      <div class="answer">
+        <p>The Box = a crossfit gym. It's called a box because it's typically in a warehouse-type space. Instead of seeing the machines you see in a typical gym, you're more likely to see stacks of weights, barbells, pull-up bars and lots of open space.</p>
+      </div>
+      <div class="logos-in-accordion">
+      <img class="logo-lift" src="./img/lift.png" alt="logo">
+      <img class="logo-lift" src="./img/kettlebell-512.png" alt="logo">
+      <img class="logo-lift" src="./img/Run.png" alt="logo">
+    </div>
+ </div>
+
+
+
+
+          <!-- 6. FOOTER-->
     </section>
-    <!-- dont forget to link the javascript file here! -->
+      <footer class="footer">
+        <p>&copy Ann-Sofi Jönsson Technigo Bootcamp 2021</p>
+        <img class="footer-logo" src="./img/sport_barbell-512.png" alt="barbell">
+      </footer>
+    </section>
```

Comment:
See if you can remove the wrapping `section` of the `footer`

Reply:
Yep, it worked and it looks the same :-)